tests/Fixtures/Integration/priority/class_attributes_separation,braces.test

```diff
@@ -0,0 +1,25 @@
+--TEST--
```

Comment:
Hi @BackEndTea thanks for picking this up!
I've trying out this integration-priority-test. The test passes which is good of course, however if I change the priority the test passes as well. For example I've changed the priority for the ClassAttributesSeparationFixer to 5500 and -5500.
This implies that this test proves the fixers work well together, but not that there is a priority issue. This last conclusion is unfortunate as that is why we write these test. Could you double-check this test?

Reply:
So...
I did double check.
This itests indeed shall be moved to `misc` tests, it doesn't belong to `priority` tests.
We do have check that make sure if fixers would be applied in reverse order the expected outcome must be the same. But we don't check if order matters for `priority` cases.

@BackEndTea , please move this itest to `misc` or extend it to prove it is a priority tests indeed.

@SpacePossum , let us move discussion to: #3444